checks.d/teamcity.py

```diff
@@ -0,0 +1,90 @@
+        if self.last_build_ids.get(instance_name, None) is None:
+            self.log.info("Initializing {}".format(instance_name))
+            request = requests.get(
+                "http://{}/guestAuth/app/rest/builds/?locator=buildType:{},count:1".format(self.server,
+                                                                                           build_configuration),
+                timeout=30, headers=self.headers)
+            if request.status_code != requests.codes.ok:
+                raise Exception("TeamCity reported error on initialization. Status code: {}".format(request.status_code))
+            last_build_id = request.json()["build"][0]["id"]
+            self.log.info("Last build id for {} is {}.".format(instance_name, last_build_id))
+            self.last_build_ids[instance_name] = last_build_id
+
+    def _build_and_send_event(self, new_build, instance_name, is_deployment, host, tags):
+        self.log.info("Found new build with id {}, saving and alerting.".format(new_build["id"]))
+        self.last_build_ids[instance_name] = new_build["id"]
+
+        output = {
+            "timestamp": int(time.time()),
+            "alert_type": "info",
+            "tags": []
+        }
+        if is_deployment:
+            output["event_type"] = "deployment"
+            output["msg_title"] = "{} deployed to {}".format(instance_name, host)
+            output["msg_text"] = "Build Number: {}\n\nMore Info: {}".format(new_build["number"], new_build["webUrl"])
+            output["tags"].append("deployment")
+        else:
+            output["event_type"] = "build"
+            output["msg_title"] = "Build for {} successful".format(instance_name)
+            output["msg_text"] = "Build Number: {}\nDeployed To: {}\n\nMore Info: {}".format(new_build["number"], host,
+                                                                                             new_build["webUrl"])
+            output["tags"].append("build")
+
+        if tags is not None:
+            output["tags"].extend(tags)
+        if host is not None:
+            output["host"] = host
+
+        self.event(output)
+
+    def check(self, instance):
+        instance_name = instance.get("name")
+        if instance_name is None:
+            raise Exception("Each instance must have a name")
+        build_configuration = instance.get("build_configuration")
+        if build_configuration is None:
+            raise Exception("Each instance must have a build configuration")
+        host = instance.get("host_affected")
+        tags = instance.get("tags")
+        is_deployment = instance.get("is_deployment")
+        if type(is_deployment) is not bool:
+            is_deployment = instance.get("is_deployment").lower() == "true"
+
+        self._initialize_if_required(instance_name, build_configuration)
+
+        request = requests.get(
+            "http://{}/guestAuth/app/rest/builds/?locator=buildType:{},sinceBuild:id:{},status:SUCCESS".format(
```

Comment:
This string formatting syntax is not compatible with python 2.6
```python
Python 2.6.9 (unknown, Sep  9 2014, 15:05:12)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.39)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> a = "{}".format("hello")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: zero length field name in format
```

Could you replace it with this syntax please ? 
```python
Python 2.6.9 (unknown, Sep  9 2014, 15:05:12)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.39)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> a = "{0}".format("hello")
```